src/pages/Login/login.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Link } from "react-router-dom";
+
+import { Header, Footer, Input } from '../../components';
+import { authBG } from '../../assets/images';
+
+import './login.css'
+
+function Login(){
+    return(
+        <div>
+            <Header />
+            <div className="flex-space-btwn flex-center space-around login-parent">
+                
+                <img className="auth-landing-img" src={authBG} alt="illustration of pets being together"/>
+                
+                <div>
+                <form className="flex-column glass-card login-form">
+                    <div className="flex-column flex-center">
+                        <h3 className='form-heading'>Login</h3>
+                        <Input type={"text"} placeholder={"Email"}/>
+                        <Input type={"password"} placeholder={"Password"}/>

```

Comment:
Nitpicking: you can directly pass the string as a prop. You don't need curly braces here.

Reply:
Sure, have changed it accordingly.